src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -1046,6 +1096,60 @@ public void getProjectChildrenConciseFilterByTagTest() {
                 """);
     }
 
+    @Test
+    public void getProjectChildrenConciseFilterByTeamTest() {
+        final var parentProject = new Project();
+        parentProject.setName("acme-app");
+        qm.persist(parentProject);
+
+        final var childProjectA = new Project();
+        childProjectA.setParent(parentProject);
+        childProjectA.setName("acme-child-app-a");
+        qm.persist(childProjectA);
+
+        final var childProjectB = new Project();
+        childProjectB.setParent(parentProject);
+        childProjectB.setName("acme-child-app-b");
+        qm.persist(childProjectB);
+
+        qm.bind(childProjectB, List.of(qm.createTeam("foo")));
```

Comment:
See comment above, teams are associated with projects like this: https://github.com/DependencyTrack/hyades-apiserver/blob/5de4be3c7ba133758859ffdfd5874ce9e90a7ff5/src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java#L333-L334

`qm.bind` currently only works for tags.